source/DataFile.cpp

```diff
@@ -103,8 +103,11 @@ void DataFile::LoadData(const string &data)
 	vector<int> whiteStack(1, -1);
 	bool fileIsSpaces = false;
 	bool warned = false;
+
+	// Ignore lines which are only comments regardless of whitespace.
+	bool lineIsCommentOnly = false;
 	size_t lineNumber = 0;
-	
```

Comment:
Unrelated whitespace change

Reply:
Okay, so what do you think should be done?

Reply:
Just restoring that tab should do it. Unfortunately I can't add it as suggestion because i commented on the deleted line. Thanks github :/

Reply:
I've restored it.